views/github-configuration.hbs

```diff
@@ -1,70 +1,123 @@
-<!doctype html>
 <html>
```

Comment:
Should probably keep the doctype

Reply:
That was an accident. Didn't mean to delete that so good catch!